etc/cargo.config.dev-opt

```diff
@@ -0,0 +1,10 @@
+[profile.release]
+opt-level = 2
+debug = false
+rpath = false
+lto = false
+debug-assertions = false
+codegen-units = 16
```

Comment:
We met performance regression before when codegen-units is larger than 1

Reply:
Yeah, I've heard that. I'll rebenchmark with the current toolchain before landing anything that affects the real build.